source/_integrations/group.markdown

```diff
@@ -33,6 +33,19 @@ group:
       - device_tracker.dad_smith
       - device_tracker.mom_smith
 ```
+When using groups.yaml file you should not have the "group:" line and group names should then start in the first column.
```

Comment:
This depends on how groups.yaml is included in the main configuration file. Also, I think the "not have the group: line" is a bit unclear. Does it mean you don't need it in your configuration.yaml file?

I think it is fine to add this, however, please make sure the build passes.